src/modules/launcher/Plugins/Microsoft.Plugin.Program/ProgramArgumentParser/InferedProgramArgumentParser.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.Plugin.Program
 {
-    public class InferedProgramArgumentParser : IProgramArgumentParser
+    public class InferredProgramArgumentParser : IProgramArgumentParser
```

Comment:
The file needs to match class name.

Reply:
Sigh. Will fix when I get to a computer (Monday). I have a separate tool for renaming files (I wrote it after the main tool – to deal with Java files, which weren't a primary language for me). I really need to just merge it with the main tool.